version.bzl

```diff
@@ -14,7 +14,7 @@
 
 # True source of AGI versions.
 # Increment these numbers immediately after releasing a new version.
-AGI_VERSION_MAJOR="1"
+AGI_VERSION_MAJOR="2"
 AGI_VERSION_MINOR="1"
```

Comment:
As Stella raised, shouldn't this be `0`?

Reply:
See my reply to Stella.